python/ray/autoscaler/aws/example-full.yaml

```diff
@@ -108,6 +108,13 @@ file_mounts: {
 #    "/path2/on/remote/machine": "/path2/on/local/machine",
 }
 
+# Files or directories to copy from the head node to the worker nodes. The format is a
+# dictionary from WORKER_NODE_PATH: HEAD_NODE_PATH, e.g.
+worker_file_mounts: {
+#    "/path1/on/worker/machine": "/path1/on/head/machine",
+#    "/path2/on/worker/machine": "/path2/on/head/machine",
+}
```

Comment:
Can this just be a list? and assume that the head node file system structure is the same as worker?

Reply:
yeah that makes sense